contracts/ConversionRates2.sol

```diff
@@ -21,8 +21,8 @@ contract ConversionRates2 is ConversionRates {
         public
```

Comment:
should find a better name
not conversionRate2

maybe just
enhancedStepFunctions

not sure.

Reply:
Changed to `EnhancedStepFunctions`